pages/sign-up.html

```diff
@@ -0,0 +1,83 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cruxui-lib.netlify.app//Components/main.css"
+    />
+    <link rel="stylesheet" href="../style.css" />
+    <script src="https://code.iconify.design/2/2.1.2/iconify.min.js"></script>
+    <title>CALYX-ECOMM</title>
+  </head>
+  <body>
+    <div class="rootContainer lightMode">
+      <!-- NavBar -->
+      <div class="navbar p-2" style="z-index: 100">
+        <input id="menu-icon" type="checkbox" />
+        <label class="menu-label" for="menu-icon"></label>
+        <div class="nav-brand">
+          <div class="h3 text-bold color-primary box-3d p-1-x bg-secondary">
+            CALYX Mart
+          </div>
+        </div>
+      </div>
+      <!-- Signup -->
+      <div class="signUp-box border-secondary box-3d p-4">
+        <div class="h5 color-secondary p-2-y">
+          Sign Up to become a Calyx user.
+        </div>
+        <label class="input-label">Email Id</label>
+        <div class="input-group">
+          <input
+            class="text-input"
+            type="text"
+            placeholder="  Enter email id..."
```

Comment:
input type will be email for email id input

Reply:
the type is being controlled by JS